src/Manifest-Core/TheManifestBuilder.class.st

```diff
@@ -524,6 +524,36 @@ TheManifestBuilder >> removeAllToDo: fp of: ruleId version: versionId [
 	self removeAllItem: fp selector: selector
 ]
 
+{ #category : #class }
+TheManifestBuilder >> removeClasses: aCollection [
+	"Remove the selected classes from the system.  Check that the user really wants to do this, since it is not reversible.  Answer true if removal actually happened."
+
+	| classNames classesToRemove result |
+	aCollection isEmptyOrNil
+		ifTrue: [ ^ false ].
+	classesToRemove := aCollection collect: [:each | each instanceSide].
+	classNames := (classesToRemove collect: [:each | each name]) joinUsing: ', '.
+	(result := self confirm: (self removeClassesMessageFor: classNames))
+		ifTrue: [
+			classesToRemove
+				do: [ :classToRemove |
+					classToRemove subclasses notEmpty
+						ifTrue: [
+							(self confirm: (self removedClassHasSubclassesMessageFor: classToRemove name))
+								ifTrue: [ classToRemove removeFromSystem ] ]
+						ifFalse: [ classToRemove removeFromSystem ] ] ].
+	^ result
+]
```

Comment:
This method is called only by a dead method so we can remove it because it's really weird for the manifest to implement some classes removal. 

Can be done in a next PR